Hyrax 1530

kyang2014 opened the pull request:
draft

jgallagher59701 commented on modules/dmrpp_module/build_dmrpp_util.cc:
```diff
@@ -392,20 +397,267 @@ get_value_as_string(hid_t h5_type_id, vector<char> &value)
-                   "This is not yet supported by the dmr++ creation machinery. ");
             string str_fv(value.begin(),value.end());
-            throw UnsupportedTypeException(msg);
+            break;
```
Here I think the value of str_fv should be returned. I think no value is returned for this case.

kyang2014 replied:
The compound datatype fill value is calculated elsewhere.  Here we need to use break only.

jgallagher59701 reviewed (approved):
Please check the code in build_dmrpp_util.cc. It might make things clearer if the #if 0 ... #endif blocks are removed.

kyang2014 commented:
> Please check the code in build_dmrpp_util.cc. It might make things clearer if the #if 0 ... #endif blocks are removed.

I removed all the #if 0 #endif blocks  I introduced in the last commit.  The other #if 0 #endif blocks may be added by other people. We can do that when we have a ticket to simple review the code.
